src/lsm/tree_fuzz.zig

```diff
@@ -34,7 +34,7 @@ const Tree = @import("tree.zig").TreeType(Table, Storage, "Key.Value");
 const Grid = GridType(Storage);
 const SuperBlock = vsr.SuperBlockType(Storage);
 
-pub const tigerbeetle_config = @import("../config.zig").configs.test_min;
+pub const tigerbeetle_config = @import("../constants.zig").configs.test_min;
```

Comment:
Should it be `@import("../config.zig")`? Currently compile errors

Reply:
Thanks! Updated via rebase.